frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -111,12 +123,59 @@ export function Surveys(): JSX.Element {
                     { key: SurveysTabs.Archived, label: 'Archived' },
                     showLinkedHogFunctions ? { key: SurveysTabs.Notifications, label: 'Notifications' } : null,
                     { key: SurveysTabs.History, label: 'History' },
+                    surveysConfigAvailable ? { key: SurveysTabs.Settings, label: 'Settings' } : null,
                 ]}
             />
-
-            {tab === SurveysTabs.History ? (
-                <ActivityLog scope={ActivityScope.SURVEY} />
-            ) : tab === SurveysTabs.Notifications ? (
+            {tab === SurveysTabs.Settings && (
+                <>
+                    <div className="flex items-center mb-2 gap-2">
+                        <div className="flex-1" />
+                        <LemonButton
+                            type="primary"
+                            onClick={() => {
+                                updateCurrentTeam({
+                                    survey_config: {
+                                        ...currentTeam?.survey_config,
+                                        appearance: {
+                                            ...currentTeam?.survey_config?.appearance,
+                                            ...editableSurveyConfig,
+                                        },
+                                    },
+                                })
+                            }}
+                        >
+                            Save settings
```

Comment:
I really want some text in this open area

<img width="962" alt="image" src="https://github.com/user-attachments/assets/6e112c2a-e7e9-4eae-b853-11e079a18150">

that tells me what the heck this page is for, because it's not clear at the moment.  I also think that changing the name from `Settings` to `Styling` or maybe `Global Styling` makes more sense.